data/transactions/logic/eval.go

```diff
@@ -3302,17 +3335,40 @@ func (cx *EvalContext) accountReference(account stackValue) (basics.Address, uin
 	}
 	idx, err := cx.Txn.Txn.IndexByAddress(addr, cx.Txn.Txn.Sender)
 
+	invalidIndex := uint64(len(cx.Txn.Txn.Accounts) + 1)
+	// Allow an address for an app that was created in group
+	if err != nil && cx.version >= createdResourcesVersion {
```

Comment:
We keep cx.created updated regardless of version, so when accessing it, we check whether we are at a version that allows in group access.